script/lib/util.py

```diff
@@ -183,15 +183,18 @@ def execute(argv, env=os.environ):  # pylint: disable=dangerous-default-value
                 argv, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
                 encoding='utf-8', universal_newlines=True)
         stdout, stderr = process.communicate()
-        if is_verbose_mode():
-            print(stdout)
-        elif process.returncode != 0:
+        if is_verbose_mode() or process.returncode != 0:
+            # Fix any unsupported characters in stdout encoder.
+            printable_stdout = stdout.encode(sys.stdout.encoding,
+                                             'backslashreplace').decode(
+                                                 sys.stdout.encoding)
             # Print the output instead of raising it, so that we get pretty output.
             # Most useful erroroutput from typescript / webpack is in stdout
             # and not stderr.
-            print(stdout)
-            raise RuntimeError('Command \'%s\' failed' %
-                               (' '.join(argv)), stderr)
+            print(printable_stdout)
+            if process.returncode != 0:
+                raise RuntimeError('Command \'%s\' failed' % (' '.join(argv)),
+                                   stderr)
```

Comment:
We're not doing the same safe encoding for `stderr` - perhaps during the `raise` python internally does the correct encoding? This is the only place chromium does their "logging" (and they also don't do the extra encoding) so perhaps that's an ok assumption.

Reply:
I checked and yes, it prints unsupported chars as `\u...` in the exception handler.